db/seeds.rb

```diff
@@ -3,57 +3,133 @@
 Category.destroy_all
 Organization.destroy_all
 
-category = Category.create!(name: 'uncategorized')
+uncategorized = Category.create!(name: "uncategorized")
+clothing = Category.create!(name: "Clothing")
+toiletries = Category.create!(name: "Toiletries")
+food_pantry_items = Category.create!(name: "Food Pantry Items")
+food = Category.create!(name: "Food")
+supplies = Category.create!(name: "Supplies")
+adults_clothing = Category.create!(name: "Adults' Clothing")
+childrens_clothing = Category.create!(name: "Childrens' Clothing")
```

Comment:
I think these variables are being assigned but not actually used anywhere.

You could actually use these in the array, instead of the string names. That would eliminate the need for all the look-ups by name later on.

Reply:
Good catch on this!